src/SFML/Audio/CMakeLists.txt

```diff
@@ -78,27 +78,24 @@ else()
     find_host_package(FLAC REQUIRED)
 endif()
 
-if(NOT SFML_OS_IOS)
-    include_directories(${OPENAL_INCLUDE_DIR})
-endif()
-include_directories(${VORBIS_INCLUDE_DIRS})
-include_directories(${FLAC_INCLUDE_DIR})
-add_definitions(-DOV_EXCLUDE_STATIC_CALLBACKS) # avoids warnings in vorbisfile.h
-add_definitions(-DFLAC__NO_DLL)
+# define the sfml-audio target
+sfml_add_library(sfml-audio
```

Comment:
target needs to be defined before using target_link_libraries(), target_include_directories() and target_compile_definitions(), so moved from below